ocm_python_wrapper/cluster.py

```diff
@@ -395,8 +395,10 @@ def install_addon(
             params_command = ""
             for parameter in parameters:
                 params_command += f" --{parameter['id']} {parameter['value']}"
+            # TODO remove support for billing-model flag once https://github.com/openshift/rosa/issues/1279 resolved
             res = rosa_cli.execute(
                 command=f"install addon {self.addon_name} --cluster {self.name} {params_command}"
+                f" --billing-model standard"
```

Comment:
is this required for all addons?

Reply:
I've checked their implementation along the addon install command flow in the rosa repository, and it doesn't validate the billing model against the specific addon being installed.
In addition, the docs don't refer to a specific addon.
Of course RHODS installation is requiring it as well.

Is there another way I should try to find out?

Reply:
From the API: `/api/clusters_mgmt/v1/clusters/{cluster_id}/addons` POST scheme:

```
  "billing": {
    "kind": "string",
    "id": "string",
    "href": "string",
    "billing_marketplace_account": "string",
    "billing_model": "marketplace"
  },
```